src/main/java/com/solr/clientwrapper/domain/port/api/DataIngectionServicePort.java

```diff
@@ -0,0 +1,6 @@
+package com.solr.clientwrapper.domain.port.api;
+
+public interface DataIngectionServicePort {	
```

Comment:
Small typo: it's 'j' in Injection. Would be better if we change it.
I have also renamed it for the part I have integrated your code in my branch.